src/native-shim.js

```diff
@@ -21,10 +21,7 @@
   let origHTMLElement = HTMLElement;
   // TODO(justinfagnani): Tests!!
   window.HTMLElement = function() {
-    // prefer new.target for elements that call super() constructors or
-    // Reflect.construct directly
-    let newTarget = new.target || this.constructor;
-    return Reflect.construct(origHTMLElement, [], newTarget);
+    return Reflect.construct(origHTMLElement, [], this.constructor);
   }
```

Comment:
I'm seeing a lint error complaining that this is an assignment, so a `;` after the closing brace is called for.